yamux/src/connection/stream.rs

```diff
@@ -161,7 +161,7 @@ impl Stream {
             conn,
             config: config.clone(),
             sender,
-            flag: Flag::None,
+            flag: Flag::Ack,
```

Comment:
Is this change correct? Doesn't this now implicitly ACK all inbound streams?

Reply:
`Stream::flag` is the flag that is piggy-backed onto the next package sent to the remote. This does not in itself trigger a new package being sent containing the acknowledgement. Thus I don't this this will "now implicity ACK all inbound streams". Am I missing something @thomaseizinger?